tools/please_go_test/gotest/write_test_main.go

```diff
@@ -208,6 +209,7 @@ func main() {
 {{end}}
     testVar := os.Getenv("TESTS")
     if testVar != "" {
+		testVar = strings.Replace(testVar, " ", "|", -1)
```

Comment:
Not sure what's wrong with indentation here I seem to be using spaces with 4-space tabs.

Reply:
I think gofmt uses tabs?

Reply:
Think this whole block is a template string thus gofmt doesn't format it. That and github uses 8-space which makes it look odd.